src/web/clients/admin/common.mli

```diff
@@ -19,6 +19,7 @@
 (*  <http://www.gnu.org/licenses/>.                                       *)
 (**************************************************************************)
 
+open Js_of_ocaml
```

Comment:
It may be unfortunate to include this here but allows to declare `read_full` below.